src/app/(nextjs_migration)/(guest)/user/unsubscribe-monthly/page.tsx

```diff
@@ -0,0 +1,34 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+import { redirect } from "next/navigation";
+import { updateMonthlyEmailOptout } from "../../../../../db/tables/subscribers";
+import { getL10n } from "../../../../functions/server/l10n";
+
+export default async function PublicScan(props: {
```

Comment:
Suggestion: We might want to rename `PublicScan` -> `UnsubscribeMontly` or similar.